samples/nextjs/sitecore/config/JssNextWeb.config

```diff
@@ -11,13 +11,20 @@
   <sitecore>
     <settings>
       <!--
-        PROXY CONFIGURATION
-        When running JSS with the headless proxy (node-headless-ssr-proxy),
-        the original IP address of clients is obscured by the proxy.
-        This setting tells Sitecore to read the forwarded header the proxy sends,
-        thus making analytics track the correct original client IP address.
+        ANALYTICS FORWARDED REQUEST HEADER
+        When using Next.js SSR routes, Layout Service requests will send the original IP address of the client on the 'X-Forwarded-For' header.
+        This setting tells Sitecore to read the forwarded header, thus making analytics track the correct original client IP address.
       -->
       <setting name="Analytics.ForwardedRequestHttpHeader" set:value="X-Forwarded-For" />
```

Comment:
Should this be commented out by default since people have to opt-in to SSR?

Reply:
@anastasiya29  Yes, I think it should. Commented out and updated config doc.